src/SCREENS.ts

```diff
@@ -318,6 +318,7 @@ const SCREENS = {
         TAG_EDIT: 'Tag_Edit',
         TAXES: 'Workspace_Taxes',
         REPORT_FIELDS: 'Workspace_ReportFields',
+        REPORT_FIELD_SETTINGS: 'Workspace_ReportField_Settings',
```

Comment:
I think we should keep this cohesive and keep all the names same so.
```suggestion
        REPORT_FIELDS_SETTINGS: 'Workspace_ReportFields_Settings',
```

If there would be other page that fits `settings` more we can rename this to `details`

Reply:
@mountiny This is a different page, we have a report_FIELD_settings -> settings for a single report field
And we have report_FIELDS_settings -> settings for all report fields